change/@internal-calling-component-bindings-53f331c0-2381-497a-b818-8491e9e0287f.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Bump calling beta dependency for unsupportedBrowser functionality.",
```

nit: for changelogs its nicer to declare the version number:
```suggestion
  "comment": "Bump `@azure/communication-calling` beta dependency to 1.9.1-beta.1",
```